owltorrent-client/build.gradle

```diff
@@ -15,4 +15,11 @@ jar {
     from {
         configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
     }
-}
\ No newline at end of file
+}
+
+test {
+    testLogging {
+        events "failed"
+        exceptionFormat "full"
+    }
```

Comment:
This should be added to the build.gradle in the root project instead. This way you don't need 4 dupes.